cli/src/serve.js

```diff
@@ -404,6 +439,10 @@ const read_config_from_flags = (parsed) => {
     config.bind = parsed.bind;
   }
 
+  if (parsed.connect_timeout !== null) {
```

Comment:
This should also make sure `parsed.connect_timeout` is not `undefined` - this function is called by the `get-schema` and `set-schema` scripts which may or may not set all of these.